src/newrelic.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names. Only applicable for local development because deployed
+   * environments use the env variable.
+   *
+   * @env NEW_RELIC_APP_NAME
+   */
+  app_name: ['tta-smart-hub-development'],
+  /**
+   * This setting controls distributed tracing.
+   * Distributed tracing lets you see the path that a request takes through your
+   * distributed system. Enabling distributed tracing changes the behavior of some
+   * New Relic features, so carefully consult the transition guide before you enable
+   * this feature: https://docs.newrelic.com/docs/transition-guide-distributed-tracing
+   * Default is true.
+   */
+  distributed_tracing: {
+    /**
+     * Enables/disables distributed tracing.
+     *
+     * @env NEW_RELIC_DISTRIBUTED_TRACING_ENABLED
+     */
+    enabled: true,
+  },
+  logging: {
+    /**
+     * Level at which to log. 'trace' is most useful to New Relic when diagnosing
+     * issues with the agent, 'info' and higher will impose the least overhead on
+     * production applications.
+     */
+    level: 'info',
+    filepath: 'stdout',
+  },
+  /**
+   * When true, all request headers except for those listed in attributes.exclude
+   * will be captured for all traces, unless otherwise specified in a destination's
+   * attributes include/exclude lists.
+   */
+  allow_all_headers: false,
+  attributes: {
+    /**
+     * Attributes to include from all destinations. Allows * as wildcard at end.
+     *
+     * NOTE: If including headers, they must be in camelCase form to be included.
+     *
+     * @env NEW_RELIC_ATTRIBUTES_INCLUDE
+     */
+    include: [
+      'request.headers.accept',
+      'request.headers.host',
+      'request.headers.referer',
+      'request.headers.userAgent',
+      'response.headers.contentType',
+      'response.headers.date',
+    ],
+    /**
+     * Prefix of attributes to exclude from all destinations. Allows * as wildcard
+     * at end. Will only have effect if `allow_all_headers` is true
+     *
+     * NOTE: If excluding headers, they must be in camelCase form to be filtered.
+     *
+     * @env NEW_RELIC_ATTRIBUTES_EXCLUDE
+     */
+    exclude: [

```

Comment:
I think it's good to keep the exclude list in place in case we decide to switch back to `allow_all_headers: true`

Reply:
Agreed, if nothing else it documents the headers we consider PII or otherwise unsafe to log